package.json

```diff
@@ -142,6 +145,7 @@
     "terra-disclosure-manager": "^4.9.0",
     "terra-enzyme-intl": "^3.4.0",
     "terra-icon": "^3.58.0",
+    "terra-table": "^5.4.0",
```

Comment:
Just curious why this needs to be added to the root package.json?

Reply:
Not sure why terra-icon is at the root level as well but I'm trying to move away from this pattern as these packages aren't used at the root level. I would rather have a linter exception than an extra package at the root level.

But this change can also be done in the monorepo.

+1 for now.